CODEOWNERS

```diff
@@ -1,11 +1 @@
 *               @gmaurel @mwoehlke-kitware @micheleCTDEAdmin
-
-.editorconfig   @mihaipopescu
-
-# CI
-.travis.yml     @mihaipopescu
-appveyor.yml    @mihaipopescu
-
-# emscripten
-#/emscripten/    @CDanU
-#/src/uncrustify_emscripten.cpp @CDanU
```

Comment:
Can you amend to avoid removing other people's name? I guess it was an oversight.

Reply:
I can separate into a different commit but those lines are commented out. We have the git history for that.

Reply:
Fair enough :-)